packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -292,15 +293,24 @@ class _AutocompleteOptions<T extends Object> extends StatelessWidget {
             itemCount: options.length,
             itemBuilder: (BuildContext context, int index) {
               final T option = options.elementAt(index);
-              final bool highlight = AutocompleteHighlightedOption.of(context) == index;
               return InkWell(
                 onTap: () {
                   onSelected(option);
                 },
-                child: Container(
-                  color: highlight ? Theme.of(context).focusColor : null,
-                  padding: const EdgeInsets.all(16.0),
-                  child: Text(displayStringForOption(option)),
+                child: Builder(
+                  builder: (BuildContext context) {
+                    final bool highlight = AutocompleteHighlightedOption.of(context) == index;
+                    if (highlight) {
+                      SchedulerBinding.instance!.addPostFrameCallback((Duration timeStamp) {
```

Comment:
I think that this makes sense, since we need to build it first to find out if it **isn't** visible and scroll it into view if that is the case.

Otherwise you might want to just initialize at the correct index.... which brings us back to https://github.com/flutter/flutter/issues/12319. That issue was resolved by the scrollable_positioned_list package, which does not help us here. 🤔